cg/models/demultiplex/run_parameters.py

```diff
@@ -299,3 +311,13 @@ def get_read_1_cycles(self) -> int:
     def get_read_2_cycles(self) -> int:
         """Return the number of cycles in the second read."""
         return self._read_parser.get(RunParametersXMLNodes.READ_2_NOVASEQ_X)
+
+    def get_flow_cell_model(self) -> str:
+        """Return flow cell model."""
+        consumable_infos = self.tree.findall(".//ConsumableInfo")
+        for consumable_info in consumable_infos:
+            type_element = consumable_info.find("Type")
+            if type_element is not None and type_element.text == "FlowCell":
+                name_element = consumable_info.find("Name") or consumable_info.find("Mode")
```

Comment:
I think the `or consumable_info.find("Mode")` is not needed, I think it will always be `"Name"`

Reply:
In fixtures dir:
```
    <ConsumableInfo>
      <SerialNumber>22F52TLT3</SerialNumber>
      <LotNumber>20775712</LotNumber>
      <PartNumber>20080370</PartNumber>
      <ExpirationDate>2025-01-10T00:00:00+01:00</ExpirationDate>
      <Type>FlowCell</Type>
      <Mode>10B</Mode>
    </ConsumableInfo>
    <ConsumableInfo>
    ```

Reply:
Hasta:
```
    <ConsumableInfo>
      <SerialNumber>22GTJKLT3</SerialNumber>
      <LotNumber>20795878</LotNumber>
      <PartNumber>20080370</PartNumber>
      <ExpirationDate>2025-03-27T00:00:00+01:00</ExpirationDate>
      <Type>FlowCell</Type>
      <Mode>3</Mode>
      <Version>1.0</Version>
      <Name>10B</Name>
    </ConsumableInfo>
```